.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Push Docker Images
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish_docker_image:
+    name: Publish Docker Image
+    runs-on: ubuntu-20.04
+    if: github.event_name == 'push'
```

Comment:
With the `on` condition above, would `event_name` ever be something other than "push"?  Seems redundant, but haven't looked into it in depth.